pkg/controller/installation/products/threescale/objects.go

```diff
@@ -0,0 +1,127 @@
+package threescale
```

Comment:
this is test related code too right? So let's add _test.go to the filename?

Reply:
Have add `_test.go` to all these